dask/dataframe/core.py

```diff
@@ -1755,6 +1756,22 @@ def astype(self, dtype):
         empty = self._pd.astype(dtype)
         return map_partitions(pd.DataFrame.astype, empty, self, dtype=dtype)
 
+    def info(self):
+        """
+        Concise summary of a Dask DataFrame.
+        """
+        from pandas.formats.format import _put_lines
+        lines = list()
+        lines.append(str(type(self)))
+        lines.append('Data columns (total %d columns):' % len(self.columns))
+        dtypes = self.dtypes
+        template = "%s    %s"
+        for i, col in enumerate(self.columns):
+            dtype = dtypes.iloc[i]
+            lines.append(template % (col, dtype))
+
+        sys.stdout.write('\n'.join(lines))
```

Comment:
Is there a reason to use `sys.stdout.write` instead of the more common `print('\n'.join(lines))`?